package.json

```diff
@@ -23,7 +23,9 @@
     "tiddlywiki5",
     "wiki"
   ],
-  "dependencies": {},
+  "dependencies": {
+    "uuid": "^3.3.2"
```

Comment:
To ensure that TW is fully self contained we don't use NPM for installing dependencies. Instead we copy the code into the core (e.g. https://github.com/Jermolene/TiddlyWiki5/tree/master/core/modules/utils/diff-match-patch)